app/services/users/push_to_rdv_solidarites.rb

```diff
@@ -98,13 +92,26 @@ def update_rdv_solidarites_user
     end
 
     def rdv_solidarites_user_attributes
-      user_attributes = @user.attributes
-                             .symbolize_keys
-                             .slice(*User::SHARED_ATTRIBUTES_WITH_RDV_SOLIDARITES)
-                             .transform_values(&:presence)
-                             .compact
-      user_attributes.delete(:email) if @user.conjoint?
-      user_attributes
+      attrs = @user.attributes
+                   .symbolize_keys
+                   .slice(*User::SHARED_ATTRIBUTES_WITH_RDV_SOLIDARITES)
+                   .transform_values(&:presence)
+                   .compact
+
+      select_appropriate_email_field(attrs)
+
+      attrs
+    end
+
+    def select_appropriate_email_field(attrs)
+      if @rdv_solidarites_user_id
+        # Existing users in RDV-S could have either email or notification_email set
+        # Only convert to notification_email if that's the field currently used
+        attrs[:notification_email] = attrs.delete(:email) if email_field == :notification_email
+      else
+        # New users are always created with notification_email in RDV-S
+        attrs[:notification_email] = attrs.delete(:email)
+      end
```

Comment:
Je pense qu'on peut faire moins compliqué ici : 

```suggestion
      attrs[:notification_email] = attrs.delete(:email) if should_target_notification_email?

      attrs
    end

    def should_target_notification_email?
      @rdv_solidarites_user_id.nil? || email_field_to_target == :notification_email
    end
```

Reply:
Effectivement 😅